requirements/base.txt

```diff
@@ -82,3 +82,8 @@ sentry-sdk
 websocket-client
 bleach
 python-magic
+rcssmin --install-option="--without-c-extensions"
+rjsmin --install-option="--without-c-extensions"
+django-compressor --upgrade
```

Comment:
Do we need the `--upgrade` option? 
An auto-upgrade might mess things up if there ever is a breaking upgrade for compressor 

Also travis fails 

```
ERROR: Invalid requirement: django-compressor --upgrade
pip: error: no such option: --upgrade
The command "pip install -r requirements/test.txt" exited with 1.
```

Reply:
Agree - having auto-upgrade is a bit scary.